backend/danswer/connectors/web/connector.py

```diff
@@ -84,6 +85,8 @@ def load_from_state(self) -> GenerateDocumentsOutput:
             logger.info(f"Indexing {current_url}")
 
             try:
+                current_visit_time = datetime.now().strftime("%B %d, %Y, %H:%M:%S")
```

Comment:
cant get page update time reliably, time visited is still relevant imo but can discuss when we add metadata to all connectors